includes/Integrations/Site_Kit.php

```diff
@@ -95,7 +95,7 @@ protected function is_plugin_active(): bool {
 	 */
 	protected function is_adsense_module_active(): bool {
 		$adsense_module_active       = in_array( 'adsense', $this->get_site_kit_active_modules_option(), true );
-		$adsense_options             = get_option( 'googlesitekit_adsense_settings' );
+		$adsense_options             = (array) get_option( 'googlesitekit_adsense_settings' );
```

Comment:
Should this use the settings object?

Reply:
The `Settings` object is used for retrieving our own plugin's settings that we've registered with `registered_setting`, We don't currently use it for other settings, especially not some from other plugins.

Rethinking that is best done in a dedicated ticket.